performance/tests/Items.test.js

```diff
@@ -0,0 +1,5 @@
+const {listItemTests} = require("./ListItemsTests");
+
+const componentName = 'Items'
+listItemTests(componentName, 12);
+listItemTests(componentName, 100);
```

Comment:
Same with above VirtualList.test.js
```suggestion
listItemTests(componentName);
```